types.go

```diff
@@ -14,3 +14,7 @@ type Result struct {
 	Measurements MeasurementsMap `json:"measurements"`
 	Message      interface{}     `json:"message"`
 }
+
+func floatToIntPercent(f float64) int{
+	return int(f*100+0.5)
```

Comment:
How does this convert a float to a percentage 🤔

Reply:
I needed to make a method because I use it a lot, and it becomes a mess.
Renamed to floatToIntPercentRoundUP

Reply:
It's fine to make it a function I just wonder about the calculation but maybe I don't really understand the incoming value.

Reply:
0.987 -> 99
0.102 -> 10